### PR TITLE
Make Conversion Timestamps Play Replay from that conversion

### PR DESCRIPTION
**Pull request by b3nd3r-ssbm**
ContentIndolentBudgie-size_restricted
sorry for the random black frames, OBS was acting strangely whenever dolphin opened during recording...

### CODE REVIEW

File: app/actions/fileLoader.js [hunk 1]
```diff
@@ -165,6 +165,25 @@ export function playFile(file) {
   };
 }
 
+export function playFileFromFrame(filePath, startFrame) {
+  return async (dispatch, getState) => {
+    if (!filePath) {
+      // TODO: Maybe show error message
+      return;
+    }
```
Comment (vinceau) [inline app/actions/fileLoader.js:173]: Probably don't need this.

File: app/domain/DolphinManager.js [hunk 2]
```diff
@@ -174,6 +174,21 @@ export default class DolphinManager extends EventEmitter {
...
+      replay: filePath,
+      startFrame: startingFrame,
+      isRealTimeMode: this.settings.isRealTimeMode || false,
+      commandId: uniqueId, // Indicates to Dolphin to play new replay
+    });
+
+    await this.writeCommFile(jsonString);
+    await this.runDolphin(true);
+  }
+
```
Comment (vinceau) [inline app/domain/DolphinManager.js:191]: Can you just have the `startFrame` be an optional parameter in the `playFile()` method? I don't think it makes sense to essentially write the same function twice.

**Review by vinceau — CHANGES_REQUESTED** [review]

File: app/domain/DolphinManager.js (hunk 2 above)
Comment (b3nd3r-ssbm) [reply app/domain/DolphinManager.js:191]: great idea, I should do that

File: app/actions/fileLoader.js [hunk 3]
```diff
@@ -165,6 +165,25 @@ export function playFile(file) {
   };
 }
 
+export function playFileFromFrame(filePath, startFrame) {
```
Comment (vinceau) [inline app/actions/fileLoader.js:168]: This entire function shouldn't be needed anymore. You should update the `playFile()` action with the new interface and remove pretty much all references to `playFileFromFrame()`.

**Review by vinceau — CHANGES_REQUESTED** [review]

File: app/actions/fileLoader.js (hunk 3 above)
Comment (b3nd3r-ssbm) [reply app/actions/fileLoader.js:168]: but `playFile()` requires a file, not a string to a file path. How do you suggest I go about either getting a `file` object from the file path or making the function support file paths?

**Review by NikhilNarayana — APPROVED** [review]

**Pull request merged**